style.css

```diff
@@ -2,3 +2,218 @@
 /*font-family: 'Montserrat', sans-serif;
 font-family: 'Pacifico', cursive;
 font-family: 'Poppins', sans-serif;*/
+*,*:before, *:after {
+    padding: 0;
+    margin: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+html, body {
+    height: 100%;
+    font-family: 'Poppins', sans-serif;
+}
+.container {
+    max-width: 1180px;
+    margin: 0px auto;
+    padding: 0px 10px;
+}
+.header {
+width: 100%;
+top: 0;
+left: 0;
+z-index: 50;
+position: fixed; 
+background-color: white;   
```

Comment:
Please have a little read about colour properties (and think whether you should call this "white" or rather #ffffff)

Reply:
> Please have a little read about colour properties (and think whether you should call this "white" or rather #ffffff)

Changed color names